app/views/planning_applications/validate_form.html.erb

```diff
@@ -41,22 +41,22 @@
     <p class="govuk-body">
       <%= link_to "View notification", validation_notice_planning_application_path(@planning_application), class: "govuk-link" %>
     </p>
+  <% else %>
+    <h2 class="govuk-heading-m">
+      Validation requests
+    </h2>
+    <p class="govuk-body govuk-!-margin-top-2">
+      <%= validation_request_summary(@planning_application.validation_requests, @planning_application) %>
+    </p>
+    <p class="govuk-body govuk-!-padding-bottom-6">
```

Comment:
same here, it's best if we can stick to GOV.UK's default spacing

Reply:
Looking at the diff it was here before but I'd vote to remove the custom spacing.

Reply:
It was likely put in to match whatever design was given to us at a point in time - given the gov uk framework gives us the benefit of bootstrap instead of having to create custom classes for a little bit of padding, it makes sense to use the gov uk classes to create it as its just taking advantage of the framework

That said styling isn't consistent at all in this project so I haven't been too dear about it - you'll notice we have style tags sprinkled across some views, then use the gov uk styling classes in some others like this one, and plain old css classes in others. Bringing consistency to it.. its a bigger conversation / piece for sure